common/council_treaties/fairxen_limitations_wip.txt

```diff
@@ -0,0 +1,36 @@
+# #############################
+# ##### TREATY OF FARIXEN #####
+# #############################
+
+# dont_build_capitals_if_on_treaty = {
```

Comment:
Let's remove everything related to farixen.

Reply:
Agreed. Also, file name has a typo :P

Reply:
And just to clarify, that's not saying we won't utilize this approach, just that since we're not getting into it currently, let's not include the files in the update